apps/client/src/components/ConclusionOutcome.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { isIE, isMobile } from "react-device-detect";
+import { useParams } from "react-router-dom";
+
+import { ComponentWrapper, HideForPrint, List, PrintButton } from "../atoms";
+import { Olo, topics } from "../config";
+import { sections } from "../config/matomo";
+import { actions, eventNames } from "../config/matomo";
+import Link from "./Link";
+import Markdown from "./Markdown";
+
+type Props = {
+  contactConclusion: { title: string; description: string };
+  needsPermit: boolean;
+  matomoTrackEvent: Function;
+};
+
+export const ConclusionOutcome: React.FC<Props> = ({
+  contactConclusion,
+  needsPermit,
+  matomoTrackEvent,
+}) => {
+  const { slug } = useParams();
+  const topic = topics.find((t) => t.slug === slug);
+  const heading = needsPermit
+    ? "U hebt een omgevingsvergunning nodig"
+    : "U hebt geen omgevingsvergunning nodig";
+
+  const handlePrintButton = () => {
+    matomoTrackEvent({
+      action: actions.DOWNLOAD,
+      name: eventNames.SAVE_CONCLUSION,
+    });
+    window.print();
+  };
+
+  const handlePermitButton = () => {
+    matomoTrackEvent({
+      action: actions.CLICK_EXTERNAL_NAVIGATION,
+      name: eventNames.APPLY_FOR_PERMIT,
+    });
+    // Open OLO in new tab/window
+    window.open(Olo.home, "_blank");
+  };
+
+  return (
+    <>
+      <ComponentWrapper marginBottom={16}>
+        <Heading forwardedAs="h2">
+          {contactConclusion?.title || heading}.
+        </Heading>
+      </ComponentWrapper>
+      {needsPermit && (
+        <Paragraph gutterBottom={16}>
+          U kunt deze vergunning aanvragen bij het landelijk omgevingsloket
+        </Paragraph>
+      )}
+      {contactConclusion?.description && (
+        <Markdown
+          eventLocation={sections.CONCLUSION}
+          source={contactConclusion.description}
+        />
+      )}
+      <HideForPrint>
+        {needsPermit && (
+          <ComponentWrapper marginBottom={32}>
+            <Button
+              color="secondary"
+              onClick={handlePermitButton}
+              type="button"
+            >
+              Vergunning aanvragen
+            </Button>
+          </ComponentWrapper>
+        )}
+        {!isIE && !isMobile && (
+          <PrintButton
+            marginBottom={32}
+            onClick={handlePrintButton}
+            variant="textButton"
+          >
+            Conclusie opslaan
+          </PrintButton>
+        )}
+      </HideForPrint>
+      {needsPermit && (
+        <>
+          <Heading forwardedAs="h2">Meer weten?</Heading>
+          <Paragraph>
+            Wilt u weten hoe de aanvraag werkt, wat de kosten zijn of waar u nog
+            meer aan moet denken als u gaat starten? Op onze pagina
+            <Link
+              variant={"inline"}
+              href={
+                "https://www.amsterdam.nl/veelgevraagd/?productid=%7B215DE049-EFA3-492D-A4B1-EDFF40E0BC51%7D#case_%7BC7581CB6-A3F4-4FCA-B631-C93EF03A108D%7D"
```

Comment:
Can you remove the hash `#case_%7BC7581CB6-A3F4-4FCA-B631-C93EF03A108D%7D` from the link?

Reply:
We need the hash to open the right section